### PR TITLE
CMOS-103 licensing review and docs

### PR DESCRIPTION
**Pull request by patrick-stephens**
Add licensing docs and make more visible post review

### CODE REVIEW

File: .github/workflows/tern-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: tern-analysis
```
Comment (markspolakovs) [inline .github/workflows/tern-analysis.yml:1]: Could we have a more descriptive name? (I didn't know what "tern" is before looking it up.)

Comment (patrick-stephens) [reply .github/workflows/tern-analysis.yml:1]: What do you mean, surely it's obvious?!

But yeah, of course.

File: docs/modules/ROOT/pages/licensing.adoc [hunk 2]
```diff
@@ -0,0 +1,106 @@
+= Licensing
+
+[abstract]
+CMOS is composed of primarily other OSS software, not linked but used directly. This page provides an overview of the relevant licensing.
+
+== Background
+
+Before diving into this page, it is worth reviewing industry and standards bodies information on container licensing.
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/licensing.adoc:8]: nit

```suggestion
Before diving into this page, it is worth reviewing industry and standards bodies' information on container licensing.
```

File: docs/modules/ROOT/pages/licensing.adoc [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+* Loki: AGPL 3.0 https://github.com/grafana/loki/blob/main/LICENSE
+* Prometheus: Apache 2.0 https://github.com/prometheus/prometheus/blob/main/LICENSE
+* Alert Manager: Apache 2.0 https://github.com/prometheus/alertmanager/blob/master/LICENSE
+* Jaeger: Apache 2.0 https://github.com/jaegertracing/jaeger/blob/master/LICENSE
+* Nginx: https://github.com/nginxinc/docker-nginx/blob/master/LICENSE
+* Prometheus Merge Tool: Apache 2.0 https://github.com/lablabs/prometheus-alert-overrider/blob/master/LICENSE
+* Alpine.js: MIT https://github.com/alpinejs/alpine/blob/main/LICENSE.md
+* Couchbase Cluster Monitor: Proprietary to Couchbase https://github.com/couchbaselabs/cbmultimanager/blob/master/LICENSE
+
+All licences are in the source repository and the microlith container in the link:https://github.com/couchbaselabs/observability/blob/main/microlith/licenses/[`/licenses`] directory.
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/licensing.adoc:25]: Which source repository - ours, or the components'?

File: docs/modules/ROOT/pages/licensing.adoc [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+A statement is printed out to standard output/console at start up to indicate acceptance of licensing and where you can find them all.
+
+A simple link:https://github.com/couchbaselabs/observability/blob/main/tools/build-oss-container.sh[helper script] is provided as well to build without any proprietary components.
+
+== Container layers
+
+The various external components are reused directly from the layers already published by the entities responsible (e.g. Grafana publish the Loki image layers): this container is not redistributing or repackaging them but using as-is.
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/licensing.adoc:33]: Aren't we redistributing them though?

File: docs/modules/ROOT/pages/licensing.adoc [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+COPY --from=loki-official /usr/bin/loki /usr/bin/loki
+COPY --from=loki-official /etc/loki /etc/loki
+
+COPY --from=prometheus-official /bin/prometheus /bin/prometheus
+COPY --from=prometheus-official /bin/promtool /bin/promtool
+COPY --from=prometheus-official /etc/prometheus/prometheus.yml /etc/prometheus/prometheus.yml
+COPY --from=prometheus-official /usr/share/prometheus/console_libraries/ /usr/share/prometheus/console_libraries/
+COPY --from=prometheus-official /usr/share/prometheus/consoles/ /usr/share/prometheus/consoles/
+----
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/licensing.adoc:68]: Have you intentionally omitted the COPY instructions from Alertmanager and Jaeger?

File: docs/modules/ROOT/pages/licensing.adoc (hunk 3 above)
Comment (markspolakovs) [reply docs/modules/ROOT/pages/licensing.adoc:25]: Ah, somehow missed that

File: docs/modules/ROOT/pages/licensing.adoc (hunk 4 above)
Comment (markspolakovs) [reply docs/modules/ROOT/pages/licensing.adoc:33]: If this is the wording Legal approves of then I'm happy, was just slightly difficult to parse.

**Pull request merged**